
# Parameter Namespace (must be the same as the node namespace)
mobility:
  # Autopilot Params
  autopilot_manager:
    ros__parameters:   
      # Regular Autopilot Params
      linear_autopilot_kp: 1.0
      linear_autopilot_kd: 0.0
      linear_autopilot_ki: 0.0
      linear_autopilot_Ts: 0.1
      linear_autopilot_limit: 1.0

      angular_autopilot_kp: 40.0
      angular_autopilot_kd: 0.0
      angular_autopilot_ki: 0.0
      angular_autopilot_Ts: 0.1
      angular_autopilot_limit: 500.0

  # Aruco Autopilot Params
  aruco_autopilot_manager:
    ros__parameters:
      aruco_linear_autopilot_kp: 1.0 # 0.1
      aruco_linear_autopilot_kd: 0.0
      aruco_linear_autopilot_ki: 0.0
      aruco_linear_autopilot_Ts: 0.1
      aruco_linear_autopilot_limit: 4.0

      aruco_angular_autopilot_kp: 60.0
      aruco_angular_autopilot_kd: 0.0
      aruco_angular_autopilot_ki: 0.0
      aruco_angular_autopilot_Ts: 0.1
      aruco_angular_autopilot_limit: 100.0
      

  # Drive Params
  drive_manager:
    ros__parameters:
      cmd_lb: 0.01  # between 0-1
      max_speed: 4.0 # m/s